src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -877,6 +879,18 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
     );
   };
 
+  public getMongoIndexTabContent = (
+    mongoIndexingPolicyComponentProps: MongoIndexingPolicyComponentProps
+  ): JSX.Element => {
+    if (userContext.authType === AuthType.AAD) {
+      if (this.container.isEnableMongoCapabilityPresent()) {
```

Comment:
Do 3.2 accounts not have the mongo capability?

Reply:
No they dont. 3.6 and 4.0 have it set.